client-sdk/ts-web/core/src/client.ts

```diff
@@ -577,39 +629,18 @@ const methodDescriptorConsensusWatchBlocks = createMethodDescriptorServerStreami
     types.ConsensusBlock
 >('Consensus', 'WatchBlocks');
 
-const methodDescriptorConsensusLightGetLightBlock = createMethodDescriptorUnary<
-    types.longnum,
-    types.ConsensusLightBlock
->('ConsensusLight', 'GetLightBlock');
-const methodDescriptorConsensusLightGetLightBlockForState = createMethodDescriptorUnary<
-    types.longnum,
-    types.ConsensusLightBlock
->('ConsensusLight', 'GetLightBlockForState');
-const methodDescriptorConsensusLightGetParameters = createMethodDescriptorUnary<
-    types.longnum,
-    types.ConsensusLightParameters
->('ConsensusLight', 'GetParameters');
-const methodDescriptorConsensusLightSubmitEvidence = createMethodDescriptorUnary<
-    types.ConsensusEvidence,
-    void
->('ConsensusLight', 'SubmitEvidence');
-const methodDescriptorConsensusLightSubmitTxNoWait = createMethodDescriptorUnary<
-    types.SignatureSigned,
-    void
->('ConsensusLight', 'SubmitTxNoWait');
-
 const methodDescriptorConsensusLightStateSyncGet = createMethodDescriptorUnary<
     types.StorageGetRequest,
     types.StorageProofResponse
->('ConsensusLight', 'StateSyncGet');
+>('ConsensusLightState', 'SyncGet');
```

Comment:
2. 
```suggestion
>('Consensus', 'StateSyncGet');
```

and similarly below. we don't currently have a way to add a prefix to the method names, so the "State" word is incorrectly tacked on to the service name here. need to change them manually

Reply:
method descriptor name above should automatically regenerate as `methodDescriptorConsensusStateSyncGet`

Reply:
Oh I see, that's why there's this "State" at the end, nice hack :grin: Will fix these, thanks!

Reply:
thanks!